frontend/app/helpers/format-duration.js

```diff
@@ -26,9 +26,12 @@ export default Helper.extend({
   ),
 
   compute([duration]) {
-    // We dont want to show the minutes as minus
-    let minutes = moment.duration(Math.abs(duration)).minutes()
     let hours = trunc(duration.as('hours'))
+    // We dont want to show the minutes as minus
+    let minutes =
+      hours === 0
+        ? duration.minutes()
+        : moment.duration(Math.abs(duration)).minutes()
```

Comment:
Just `let minutes = moment.duration(Math.abs(duration)).minutes()` should work aswell since `Math.abs` does not change anything if it already is a positive number

Reply:
Is there a test for this?

Reply:
yes there is a test for this. i need the minutes in negative if something like this is formatted: `0 Hours -35 Minutes`.  And if hours is more / less than 0 it would look like this `- 3 Hours 12 Minutes`. in this case i dont need the minus from the minutes.

Reply:
Oh I see. :+1:

Reply:
Add another test for e.g "-1 Hours 3 Minutes" aswell if you haven't already.

Reply:
a test for this case exists already